crates/sui-types/src/transaction.rs

```diff
@@ -2016,7 +2016,7 @@ impl TransactionDataAPI for TransactionDataV1 {
 impl TransactionDataV1 {}
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
-pub struct SenderSignedData(Vec<SenderSignedTransaction>);
+pub struct SenderSignedData(pub Vec<SenderSignedTransaction>);
```

Comment:
is this necessary?

Reply:
Oh yeah this is no longer necessary. Was previously trying to drop a transaction to create nondeterminism, but opted for something simpler. Will remove